Lib/importlib/_bootstrap_external.py

```diff
@@ -314,6 +314,7 @@ def _write_atomic(path, data, mode=0o666):
 #     Python 3.10a2 3432 (Function annotation for MAKE_FUNCTION is changed from dict to tuple bpo-42202)
 #     Python 3.10a2 3433 (RERAISE restores f_lasti if oparg != 0)
 #     Python 3.10a6 3434 (PEP 634: Structural Pattern Matching)
+#     Python 3.10a6 3435 Use instruction offsets (as opposed to byte offsets).
```

Comment:
a7?

Reply:
Depends on how long it takes to get this merged :smile: 
We are in alpha 6 now.

Reply:
No, alpha6 was already release. The next one is alpha7:
* https://www.python.org/downloads/release/python-3100a6/
* https://www.python.org/dev/peps/pep-0619/